.github/workflows/conda-package.yml

```diff
@@ -260,11 +259,16 @@ jobs:
           python -c "import dpnp, dpctl; dpctl.lsplatform()"
           python -c "import dpnp; print(dpnp.__version__)"
 
+      - name: Set tests_path for installed package
+        run: |
+          tests_path=$(python -c "import site; import os; print(os.path.join(site.getsitepackages()[0], 'dpnp', 'tests'))")
+          echo "tests_path=$tests_path" >> $GITHUB_ENV
+
       - name: Run tests
         if: env.RERUN_TESTS_ON_FAILURE != 'true'
         run: |
           python -m pytest -q -ra --disable-warnings -vv ${{ env.TEST_SCOPE }}
-        working-directory: ${{ env.tests-path }}
+        working-directory: ${{ env.tests_path }}
```

Comment:
What about to use `python -m pytest -v --pyargs ...` here rather than specifying working directory?